code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -131,7 +131,12 @@
 	var_changes = list("burn_mod" = 0.6)
 
 //YW ADDITIONS START
-	
+/datum/trait/chem_tolerance
+	name = "Chem Tolerance"
```

Comment:
```suggestion
	name = "Major Chem Tolerance"
```
Might want to change it to that in case you plan on adding lower tiers of this trait.

Reply:
Honestly the traits as they are are really just examples. I'm open to ideas on how to actually set them out.

Thinking it might be worth splitting chem_strength in two, one var for healing and one var for drugs/toxins.

Reply:
That might be a neat idea.